app/main.py

```diff
@@ -368,11 +337,16 @@ async def forms(
             essay="This form does not exist.",
         )
 
-
-    # Get data from DynamoDB
-    user_data = table.get_item(Key={"id": user_jwt.get("id")}).get("Item", None)
-
+    # Get data from SqlModel
+    statement = (
+        select(UserModel)
+        .where(UserModel.id == user_jwt["id"])
```

Comment:
Handling of `KeyError`?

Reply:
Should not occur @Authentication.member will fail if the user has an invalid JWT